src/Entity/IdeasWorkshop/Vote.php

```diff
@@ -7,10 +7,28 @@
 use AppBundle\Entity\Adherent;
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Serializer\Annotation as SymfonySerializer;
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
- * @ApiResource
+ * @ApiResource(
+ *     attributes={
+ *         "normalization_context": {
+ *             "groups": {"vote_read"}
+ *         },
+ *         "denormalization_context": {
+ *             "groups": {"vote_write"}
+ *         },
+ *     },
+ *     collectionOperations={
+ *         "get",
+ *         "post": {"access_control": "is_granted('IS_AUTHENTICATED_FULLY')"}
```

Comment:
what about `is_granted({'ROLE_ADHERENT', 'ROLE_ADMIN_DASHBOARD'})`?
as `IS_AUTHENTICATED_FULLY` can be a simple user.

Reply:
Done.